app-frontend/src/app/pages/projects/edit/annotate/annotate.controller.js

```diff
@@ -239,46 +242,31 @@ export default class AnnotateController {
             this.annoToExport.features.push({
                 'properties': {
                     'id': id,
```

Comment:
one thing we'll have to be careful about is letting people think that these ids are unique - they have a much higher chance of collision than uuids do. For example, if an organization has a lot of people annotating a project at the same time, then compiles the exported geojson into a single one, we probably don't want them to run into a situation where the ids collide.

The safest solution would probably be to just not include ids in the exported data, since they're only used by us as something to do list tracking on.

If we do that, we might as well also just use an incrementing counter as the id, similar to the way leaflet keeps track of its layers, since that would avoid collisions while trying to generate ids for the shapes when importing